web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -219,16 +203,14 @@ export class LayerDialogComponent implements OnDestroy {
     );
   }
 
-  convertQuestionToField(fieldId: string, question: Question): Field {
-    switch (question.fieldTypeOption.type) {
+  convertQuestionToField(fieldId: string, question: Field): Field {
```

Comment:
This is a bit confusing now - it accepts a Field arg called `question`, and returns.. a field?

Reply:
this method is no longer needed as it is already field type, removed this method